ast2vec/repo2/base.py

```diff
@@ -171,10 +186,18 @@ class RepoTransformer(Transformer):
     WORKER_CLASS = None
     DEFAULT_NUM_PROCESSES = 2
 
-    def __init__(self, num_processes=DEFAULT_NUM_PROCESSES, **args):
+    def __init__(self, num_processes=DEFAULT_NUM_PROCESSES, decompose_output=False, **args):
+        """
+        Base class for transformers from repository to WORKER_CLASS model
+        :param num_processes: Number of parallel processes to transform
+        :param decompose_output: Expand output path by subfolders using the first char of \
```

Comment:
I don't like the name... `organize_files`

\+ make it an integer which sets the number of levels - not only the single one. By default is 0